frontend/src/I18n.elm

```diff
@@ -0,0 +1,44 @@
+module I18n exposing (..)
```

Comment:
it stands for "internationalization", it's a standard name for the functionality that does translation of user-facing text, see also [Internationalization_and_localization](https://en.wikipedia.org/wiki/Internationalization_and_localization#Naming). 

I'll put that link in there